source/_integrations/wallbox.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Wallbox
+description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
+ha_category:
+  - Car
+ha_release: 2021.4
+ha_iot_class: Cloud Polling
+ha_domain: wallbox
+ha_platforms:
+  - sensor
+---
+
+This `wallbox` sensor platform pulls data from https://my.wallbox.com for your wallbox charging station.
+
+## Configuration
+
+To enable this sensor in your installation, go to Configuration and Integrations and add a MyWallbox integration. You will need to fill in your wallbox:
+- serial number
+-  your username (email)
+-  password
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```

Reply:
@klaasnicolaas, thnx for the review. I included your proposals